yt_dlp/extractor/booyah.py

```diff
@@ -0,0 +1,90 @@
+from .common import InfoExtractor
+from ..utils import int_or_none, str_or_none, traverse_obj
+
+
+class BooyahBaseIE(InfoExtractor):
+    _BOOYAH_SESSION_KEY = None
+
+    def _real_initialize(self):
+        if not BooyahBaseIE._BOOYAH_SESSION_KEY:
+            initial_request = self._request_webpage(
+                'https://booyah.live/api/v3/auths/sessions', None, data="".encode())
+            BooyahBaseIE._BOOYAH_SESSION_KEY = initial_request.getheader('booyah-session-key')
+
+    def _get_comment_data(self, video_id):
```

Comment:
```suggestion
    def _get_comments(self, video_id):
```
Use `__post_extractor` for comments. See BannedVideoIE for example